department-of-veterans-affairs/va.gov-cms
# VACMS 2456 media list videos content type

## Description

See _issueid_. https://app.zenhub.com/workspaces/vagov-cms-team-5c0e7b864b5806bc2bfc2087/issues/department-of-veterans-affairs/va.gov-cms/2456

## Testing done


## Screenshots


## QA steps

As an admin
1. Login
1. Navigate here: admin/structure/types/manage/media_list_videos
3. Confirm the following AC
 - [ ] content type "Media list - Videos" | media_list_videos exists
    - [ ] add field "Alert" field_alert | entity revisions reference to "Alert (Single)" paragraph only | cardinality single | 
           optional directly under "Title and introduction" fieldset
    - [ ] add field_media_list_videos below CTA buttons field | label "Videos" | entity reference to paragraph type "Media 
           list - Videos" | cardinality - single
    - [ ] editorial / node view experience is using established patterns and does not have UX regressions
    - [ ] spec tool / tests update
4. Regression test the Alert Single component field on the Media List - Videos content type as well as the Step-by-step content type.
  - [ ] Alert single behaviors still work.

## Definition of Done
